src/librustc/session/config.rs

```diff
@@ -841,6 +844,10 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
                       `hir,typed` (HIR with types for each node).",
                      "TYPE"),
         opt::opt_u("", "show-span", "Show spans for compiler debugging", "expr|pat|ty"),
+        opt::include_u("", "include-arguments-from",
+                       "Read additional arguments from the given FILE or the \
+                        standard input if FILE is `-`. Arguments must be \
+                        separated by \\0.", "FILE", b'\0'),
```

Comment:
`\0` was mentioned by @huonw in rust-lang/cargo#1241, which is why I choose to use it, but I can change that easily if required.